tests/assets/configs/bayes.yaml

```
@@ -0,0 +1,32 @@
+- node_type: embedding
+  target_metric: retrieval_hit_rate
+  search_space:
+    - module_name: retrieval
+      k: [10]
+      embedder_name:
+        - sentence-transformers/all-MiniLM-L6-v2
+        - avsolatorio/GIST-small-Embedding-v0
+- node_type: scoring
+  target_metric: scoring_roc_auc
+  search_space:
+    - module_name: knn
+      k:
+        type: "int"
+        content:
+          low: 5
+          high: 10
+          step: 1
+      weights: [uniform, distance, closest]
+    - module_name: linear
+- node_type: decision
+  target_metric: decision_accuracy
+  search_space:
+    - module_name: threshold
+      thresh:
+        type: float
+        content:
+          low: 0.1
+          high: 0.9
```

Мб как-то так передавать `n_trials`?
```suggestion
    - module_name: threshold
     n_trials: 10
      thresh:
        type: float
        content:
          low: 0.1
          high: 0.9
```

---

Если ты сможешь обработать этот случай в валидации серч спейса то я только за

---

Есть ещё параметр n_startup_trials который очень важен